app/controllers/users_controller.rb

```diff
@@ -67,17 +69,17 @@ def user_params
     end
   end
 
-  def require_admin
-    unless current_user.admin?
-      set_flash_message :alert, :must_be_admin
-      redirect_to root_path
-    end
-  end
-
   def remove_unused_password_pair_from_params
     if params[:user][:password].blank?
       params[:user].delete(:password)
       params[:user].delete(:password_confirmation)
     end
   end
+
+  def require_admin
```

Comment:
Just saw this. If you've got this as a before filter, you don't need the logic in `#edit`